src/main/java/org/apache/solr/benchmarks/BenchmarksMain.java

```diff
@@ -94,14 +139,32 @@ public static void runQueryBenchmarks(List<QueryBenchmark> queryBenchmarks, Stri
 		            client.close();
 		        }
 
+		        Map<String, Number> validationResults = null;
+		        if (listener instanceof ValidationListener) {
+		        	try {
+		        		validationResults = runQueryValidations(benchmark.validations, ((ValidationListener) listener).queryResponses, benchmark, collection, baseUrl);
+		        	} catch (Exception e) {
+		        		log.error("Problem during validations", e);
```

Comment:
Should we just let the exception throws as is? The method signature can actually be `throws Exception` now. There was a change that wraps call these tasks into a Callable, which it can throws exception (instead of Runnable that does not allow checked exception).

If exception is thrown, then the executor should properly handle the correct task dependency (do not proceed with dependant tasks) and error reporting (print to console with the root cause)